### PR TITLE
Add a multi-admin-unit buildout.

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR adds a buildout to setup a multi-admin development environment.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -211,6 +211,73 @@ with these contents:
 respective user in our development LDAP tree.
 
 
+Setting up a multi-admin environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you need a multi-admin environment, make sure the basic development dependencies above are satisifed and run the following steps:
```
Comment (deiferni) [inline README.rst]: typo: satisifed

File: README.rst [hunk 2]
```diff
@@ -211,6 +211,73 @@ with these contents:
...
+
+Pleace notice that the default database-name for multi-admin environment is ``opengever-multi-admin``
```
Comment (deiferni) [inline README.rst]: notice -> note

**Comment by Rotonen** [comment]
`GeverSetupException: Missing Group 'og_demo-ftw_users' while creating Org-Unit 'rk'`

Are there some LDAP setup steps I'm missing for testing this out?

**Comment by phgross** [comment]
LGTM.

@Rotonen it seems to me that you didn't unselect the `Development mode` (see https://github.com/4teamwork/opengever.core/pull/3066/files#diff-88b99bb28683bd5b7e3a204826ead112R243), could that be possible?

**Comment by Rotonen** [comment]
@phgross that was it, my mistake.

What am I now missing to see when installing the second one and it cannot do an `sync_ogds`?

``` 
  Module opengever.ogds.base.sync.ogds_updater, line 65, in sync_ogds
  Module opengever.ogds.base.sync.import_stamp, line 46, in set_remote_import_stamp
  Module opengever.base.request, line 78, in dispatch_request
  Module opengever.ogds.base.utils, line 127, in get_current_admin_unit
  Module opengever.ogds.models.service, line 60, in fetch_admin_unit
  Module sqlalchemy.orm.query, line 864, in get
  Module sqlalchemy.orm.query, line 875, in _get_impl
TypeError: object of type 'NoneType' has no len()
```

Development mode and purge SQL are unchecked for the installation of the second one.

File: README.rst [hunk 3]
```diff
@@ -211,6 +211,73 @@ with these contents:
...
+    $ bin/instance fg
+
+Go to ``http://localhost:8080/manage_main`` and click on ``Install OneGov GEVER``,
+
+For the first admin-unit choose the following settings:
+
++----------------------------------+--------------------------------------+
+| Property                         | Value                                |
++==================================+======================================+
+| Deployment profile               | Choose the **first DEV-deployment**  |
```
Comment (phgross) [inline README.rst]: first DEV-deployment  --> Finanzdirektion (FD) (DEV)

File: README.rst [hunk 4]
```diff
@@ -211,6 +211,73 @@ with these contents:
...
++----------------------------------+--------------------------------------+
+| Purge SQL                        | **True**                             |
++----------------------------------+--------------------------------------+
+
+For the second admin-unit choose the following settings:
+
++----------------------------------+--------------------------------------+
+| Property                         | Value                                |
++==================================+======================================+
+| Deployment profile               | Choose the **second DEV-deployment** |
```
Comment (phgross) [inline README.rst]: second DEV-deployment --> Ratskanzlei (RK) (DEV)

File: README.rst [hunk 5]
```diff
@@ -211,6 +211,73 @@ with these contents:
...
++----------------------------------+--------------------------------------+
+| LDAP configuration profile       | OneGovGEVER-Demo LDAP                |
++----------------------------------+--------------------------------------+
+| Import users from LDAP into OGDS | True                                 |
```
Comment (phgross) [inline README.rst]: Does not work when setting up the second client, but is also not necessary --> False

Comment (phgross) [inline README.rst]: @Rotonen that was the reason for your problem you described above ...

**Comment by elioschmutz** [comment]
@phgross 👍  Fixed the readme

**Pull request merged**